### DISCUSSION TITLE
Image disappears after completion from preview pane

### DISCUSSION THREAD

**Discussion by Weatherby43** [lllyasviel/stable-diffusion-webui-forge#1695] [Q&A]
Is there a simple instruction for how to keep this from happening? The image shows all through the generation, but when it finishes, it goes to a broken image link. I believe I saw someone mention it before, but the instructions to fix it seemed a bit... convoluted. Can someone offer some easy to follow instructions please?

**Comment by lunar-studio** [comment]
I posted a bug for this recently:
https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1661
